.github/workflows/maven_build_itself.yml

```diff
@@ -23,8 +23,8 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macOS-latest]
-        java: [8, 11, 17]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Why not osx?

Reply:
This is a secondary check (verifies Maven SNAPSHOT can build itself), doesn't run the full IT suite. I asked [on Slack](https://the-asf.slack.com/archives/C7Q9JB404/p1641824261011800) a couple of days ago and it seemed nobody cared for this particular check to run on OSX. [We do run the full IT's on OSX, though.](https://github.com/apache/maven/blob/master/.github/workflows/maven.yml#L53).